### PR TITLE
Update it_it.json

### PR DESCRIPTION
**Pull request by ghost**
Updated italian translations

### CODE REVIEW

File: src/main/resources/assets/lambdynlights/lang/it_it.json [hunk 1]
```diff
@@ -1,18 +1,19 @@
 {
-  "lambdynlights.menu.light_sources": "Fonti luminose dinamiche",
+  "lambdynlights.menu.light_sources": "Sorgenti di luce Dinamica",
   "lambdynlights.menu.title": "Impostazioni di LambDynamicLights",
-  "lambdynlights.menu.canvas.1": "È raccomandato disattivare le luci dinamiche delle entità con Canvas!",
-  "lambdynlights.menu.canvas.2": "Attualmente la luminosità dinamica è molto pesante con Canvas!",
-  "lambdynlights.option.light_sources.block_entities": "Entità blocco",
+  "lambdynlights.menu.canvas.1": "É consigliato disabilitare la luce dinamica con Canvas!",
```
Comment (ZtereoHYPE) [inline src/main/resources/assets/lambdynlights/lang/it_it.json:4]: The wrong accent is used here for the verb, "È" is the correct one

File: src/main/resources/assets/lambdynlights/lang/it_it.json [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
-  "lambdynlights.tooltip.water_sensitive": "Attiva il controllo delle fonti di luce sensibili all'acqua. Questo vuol dire che certi oggetti non emetteranno luce mentre sommersi."
-}
\ No newline at end of file
+  "lambdynlights.option.light_sources.water_sensitive_check": "Controllo della sensibilità in acqua",
+  "lambdynlights.option.mode": "Luci Dinamiche",
+  "lambdynlights.tooltip.block_entities": "Abilita le luci dinamiche dei block entities",
+  "lambdynlights.tooltip.creeper_lighting": "Imposta la modalità di luce dinamica del Creeper.\\n- %s disabilita la luce dinamica del Creeper.\\n- %s imposta la luce dinamica ad un livello constante.\\n- %s imposta un livello di illuminazione dinamico.",
+  "lambdynlights.tooltip.entities": "Attiva la luce dinamica delle entità. Nota: i giocatori sono sempre attivi.",
+  "lambdynlights.tooltip.mode.1": "Attiva le luci dinamiche. Se un giocatore ha una torcia in mano illuminerà l'area per esempio.",
+  "lambdynlights.tooltip.mode.2": "%s e %s laggeranno un po' ma minimizzano gli update di luce.",
```
Comment (ZtereoHYPE) [inline src/main/resources/assets/lambdynlights/lang/it_it.json:15]: Why is update not translated? I also feel like lag here implies that the game will run slower, whereas the "delay" that was previously written made clear that lag stood for a delay in the dynamic lighting...

File: src/main/resources/assets/lambdynlights/lang/it_it.json [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
+  "lambdynlights.tooltip.mode.3": "%s è lo smooth lighting dinamico.",
```
Comment (ZtereoHYPE) [inline src/main/resources/assets/lambdynlights/lang/it_it.json:16]: Vanilla translates smooth lightning as "Illuminazione soffusa", I see no reason to use the english words here.

File: src/main/resources/assets/lambdynlights/lang/it_it.json [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
+  "lambdynlights.tooltip.tnt_lighting": "Imposta la modalità della luce dinamica della TNT.\\n- %s disabilita la luce dinamica della TNT.\\n- %s imposta la luce ad un livello costante\\n- %s imposta la luce ad un livello dinamico",
+  "lambdynlights.tooltip.water_sensitive": "Attiva il controllo della sensibilità in acqua. Significa che alcuni oggetti non faranno luce se sommersi sott'acqua."
```
Comment (ZtereoHYPE) [inline src/main/resources/assets/lambdynlights/lang/it_it.json:18]: "fare luce" feels worse than "emettere luce" (equivalent to "do light" vs "emit light")
The "se sommersi" might also imply that the objects will stop emitting light even after they have been removed from water, whereas the "mentre sommersi" implied that they will not emit *during* the immersion.

File: src/main/resources/assets/lambdynlights/lang/it_it.json [hunk 5]
```diff
@@ -1,18 +1,19 @@
 {
-  "lambdynlights.menu.light_sources": "Fonti luminose dinamiche",
+  "lambdynlights.menu.light_sources": "Sorgenti di luce Dinamica",
   "lambdynlights.menu.title": "Impostazioni di LambDynamicLights",
-  "lambdynlights.menu.canvas.1": "È raccomandato disattivare le luci dinamiche delle entità con Canvas!",
-  "lambdynlights.menu.canvas.2": "Attualmente la luminosità dinamica è molto pesante con Canvas!",
-  "lambdynlights.option.light_sources.block_entities": "Entità blocco",
+  "lambdynlights.menu.canvas.1": "É consigliato disabilitare la luce dinamica con Canvas!",
+  "lambdynlights.menu.canvas.2": "La luce dinamica con Canvas diminuisce molto la performance!",
```
Comment (ENDERZOMBI102) [inline src/main/resources/assets/lambdynlights/lang/it_it.json:5]: shouldn't this be plural?

File: src/main/resources/assets/lambdynlights/lang/it_it.json [hunk 6]
```diff
@@ -1,18 +1,19 @@
...
-  "lambdynlights.tooltip.mode.1": "Attiva le luci dinamiche. Se attivo, per esempio, un giocatore con una torcia in mano emetterà luce.",
-  "lambdynlights.tooltip.mode.2": "%s e %s avranno un leggero ritardo ma minimizzeranno gli aggiornamenti della luce.",
-  "lambdynlights.tooltip.mode.3": "%s attiva l'illuminazione dinamica soffusa.",
-  "lambdynlights.tooltip.tnt_lighting": "Imposta la modalità d'illuminazione dinamica della TNT.\n- %s disattiva l'illuminazione dinamica della TNT.\n- %s imposta una luminosità costante.\n- %s imposta una luminosità dinamica.",
-  "lambdynlights.tooltip.water_sensitive": "Attiva il controllo delle fonti di luce sensibili all'acqua. Questo vuol dire che certi oggetti non emetteranno luce mentre sommersi."
-}
\ No newline at end of file
+  "lambdynlights.option.light_sources.water_sensitive_check": "Controllo della sensibilità in acqua",
+  "lambdynlights.option.mode": "Luci Dinamiche",
+  "lambdynlights.tooltip.block_entities": "Abilita le luci dinamiche dei block entities",
```
Comment (ENDERZOMBI102) [inline src/main/resources/assets/lambdynlights/lang/it_it.json:11]: this should probably be using the female/plural version, "delle"

**Pull request closed without merging**